trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/chedd/commodities/CommodityIntendedForMapper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.defra.stw.mapping.toipaffs.chedd.commodities;
+
+import java.util.Map;
+import org.springframework.stereotype.Component;
+import uk.gov.defra.stw.mapping.dto.IncludedSpsClause;
+import uk.gov.defra.stw.mapping.dto.SpsCertificate;
+import uk.gov.defra.stw.mapping.dto.TextType;
+import uk.gov.defra.stw.mapping.toipaffs.Mapper;
+import uk.gov.defra.tracesx.notificationschema.representation.enumeration.CommodityIntention;
+
+@Component
+public class CommodityIntendedForMapper implements Mapper<SpsCertificate, CommodityIntention> {
+  private static final Map<String, CommodityIntention> COMMODITY_INTENTION_MAP = Map.ofEntries(
```

Comment:
Can do `Map.of()` instead and then the lines below just the `Map.entry`

Reply:
It won't let me do this i think due to the number of entries